src/Intervals.jl

```diff
@@ -10,8 +10,10 @@ using Dates: AbstractDateTime, value, coarserperiod
 import Base: ⊆, ⊇, ⊈, ⊉, union, union!, merge
 
 abstract type Bound end
-struct Closed <: Bound end
-struct Open <: Bound end
+abstract type Bounded <: Bound end
+struct Closed <: Bounded end
+struct Open <: Bounded end
+struct Unbounded <: Bound end
```

Comment:
Maybe would be good to have a brief explanation in the README

Reply:
Added in https://github.com/invenia/Intervals.jl/pull/104/commits/c40d64ce3693aada0ae465fb39a008dfc8931cb0